### ISSUE TITLE
Error: Package imagesLoaded not found

### ISSUE THREAD

**Issue by soundarapandian** [gmurphey/ember-masonry-grid#4]
I tried to install the addon. I am facing this issue.

```
ember install:addon ember-masonry-grid
version: 0.2.0
A new version of ember-cli is available (0.2.1). To install it, type ember update.
Installing browser packages via Bower...
  cached git://github.com/desandro/masonry.git#3.2.2
Package imagesLoaded not found
Error: Package imagesLoaded not found
```

**Comment by gmurphey** [comment]
This is caused by the blueprint file for the addon (https://github.com/gmurphey/ember-masonry-grid/blob/master/blueprints/ember-masonry-grid/index.js#L10). The package should be called `imagesloaded`.

Unfortunately, I am out of the country without a laptop until Sunday night. When I am back, I'll push a fix as soon as I can. I'm very sorry I overlooked this.

**Comment by gmurphey** [comment]
I've updated the master branch with the fix (#5). I won't be able to push a new version to npm until Sunday night, but you should be able to use the addon in the meantime by referencing the master branch in your `package.json`.

```
# Remove ember-masonry-grid from your package.json first
npm install gmurphey/ember-masonry-grid#master --save-dev
ember generate ember-masonry-grid
```

And you should be good to go after that.

Once I've released 0.3.2 to npm, I will update this issue.

**Comment by soundarapandian** [comment]
@gmurphey :+1:

**Comment by gmurphey** [comment]
I've released 0.3.2 to npm. You should be able to use `ember install:addon ember-masonry-grid` now.
